repose-aggregator/tests/release-verification/src/config/scripting.cfg.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scripting xmlns="http://docs.openrepose.org/repose/scripting/v1.0"
+           language="groovy">
+    if (request.getHeader('X-Auth-Token') == null) {
+        request.addHeader('X-Auth-Token', '12345678901234567890')
+    }
+</scripting>
```

Comment:
Yes, this is concerning.  Have you verified that the filters are being called in the logs?